.github/workflows/prune.yml

```diff
@@ -49,8 +49,8 @@ jobs:
           # replace all whitespaces with newline (required for sort) and then sort all release versions while removing any duplicates
           all_release_versions_sorted=$(echo "${all_release_versions[@]}" | tr ' ' '\n'| sort -u )
           
-          # replace the newlines with commas and then remove the trailing comma
-          all_release_versions=$(echo "${all_release_versions_sorted[@]}" | tr '\n' ',' | sed 's/,$//')
+          # remove any blank rows, replace the newlines with commas, then remove the trailing comma
+          all_release_versions=$(echo "${all_release_versions_sorted[@]}" | sed -r '/^\s*$/d'| tr '\n' ',' | sed 's/,$//')
```

Comment:
Q: Can you not just use a `-z` test of `release_versions` on line 46?

Reply:
@ewilkins-csi similar to the `-z` zero length check, we could use the `-n` not empty check and loop through the `release_versions` array
```
  # Iterate over each element in release_versions
  for version in "${release_versions[@]}"; do
      # Check if the element is not empty
      if [ -n "$version" ]; then
          # Append the element to all_release_versions
          all_release_versions+=("$version")
      fi
  done
```
in our case,  I think adding the command to remove blank rows at the end is fine since we don't have that many docker images